### PR TITLE
CONN-5002 | Release hl7 mllp 2.1.3

### PR DESCRIPTION
**Pull request by sms-ggonzalez**
Added new parameters description.

### CODE REVIEW

File: modules/ROOT/pages/hl7/hl7-mllp-connector.adoc [hunk 1]
```diff
@@ -83,12 +83,13 @@ For an advanced connection, complete these fields:
...
+** Keep connection open after data sending to be ready for next operations: Whether or not to close socket connections after each send operation.
 
-In the TLS tab you can configure a new TLS configuration, its presence implies the use of SSLServerSocket instead of plain TCP.
+In the Security tab you can configure a new TLS configuration, its presence implies the use of SSLServerSocket instead of plain TCP.
 
-image::hl7/hl7-mllp-tls.png[Studio Global Element Properties - TLS Tab]
+image::hl7/hl7-mllp-tls.png[Studio Global Element Properties - Security Tab]
 
-You can also set the send or receive buffer size.
+In the Buffer Configuration tab you can set the send or receive buffer size and set the Validate output param in order to perform special character validation.
```
Comment (kevintroller) [inline modules/ROOT/pages/hl7/hl7-mllp-connector.adoc]: param > parameter

File: modules/ROOT/pages/hl7/hl7-mllp-connector-reference.adoc [hunk 2]
```diff
@@ -90,6 +90,8 @@ Configuration for requester sockets.
 | Keep Alive a| Boolean |  Enables SO_KEEPALIVE behavior on open sockets. This automatically checks socket connections that are open but unused for long periods and closes them if the connection becomes unavailable. This is a property on the socket itself and is used by a server socket to control whether connections to the server are kept alive before they are recycled. |  false |
 | Fail On Unresolved Host a| Boolean |  Whether the socket should fail during its creation if the host set on the endpoint cannot be resolved. However, it can be set to false to allow unresolved hosts (useful when connecting through a proxy). |  true |
 | Connection Timeout a| Number |  Number of milliseconds to wait until an outbound connection to a remote server is successfully created. Defaults to 30 seconds. |  30000 |
+| Keep connection open after data sending to be ready for next operations | Boolean | Whether or not to close socket connections after each send operation. | true |
+| Validate output | Boolean | If a set throws an exception where the payload contains special characters (only ASCII chars greater than 0x1F are valid except 0x0A and 0x0D). | false |
```
Comment (kevintroller) [inline modules/ROOT/pages/hl7/hl7-mllp-connector-reference.adoc:94]: Validate output > Validate Output

**Review by kevintroller — APPROVED** [review]
LGTM, minor comments

**Review by mrjhonyvidal — APPROVED** [review]

**Comment by mcherem** [comment]
This work will not be completed by Dev. Superceded by a later release.

**Pull request closed without merging**